test-framework/sudo-test/src/lib.rs

```diff
@@ -304,24 +314,24 @@ impl EnvBuilder {
                     .unwrap()
                     .assert_success()
                     .unwrap();
-                Command::new("chmod")
-                    .arg("4775")
-                    .arg("/usr/local/bin/sudo")
-                    .arg("/usr/bin/su")
-                    .output(&env)
-                    .unwrap()
-                    .assert_success()
-                    .unwrap();
-            } else {
-                Command::new("chmod")
-                    .arg("4775")
-                    .arg("/usr/bin/sudo")
-                    .arg("/usr/bin/su")
-                    .output(&env)
-                    .unwrap()
-                    .assert_success()
-                    .unwrap();
             }
+
+            Command::new("chmod")
+                .arg("4755")
+                .arg(BIN_SUDO)
+                .arg("/usr/bin/su")
+                .output(&env)
+                .unwrap()
+                .assert_success()
+                .unwrap();
+
+            Command::new("chmod")
+                .arg("755")
+                .arg("/usr/local/sbin")
+                .output(&env)
+                .unwrap()
+                .assert_success()
+                .unwrap();
```

Comment:
`/usr/local/sbin` is only accessible by root by default in the docker images we use even though it is accessible by everyone in a regular FreeBSD install.